tests/test_occ_powercap.robot

```diff
@@ -130,6 +130,7 @@ Get Chassis URI
 
 Host PowerOn
     [Documentation]     Keyword to poweron the host and wait till boot up
+    [Tags] host-power
```

Comment:
Host PowerOn is a prereq keyword which I'm running as testsuite setup and needed for occ tests.

If host poweron is not supported on the qemu based platform then we should have a way to find out the current running platform and return from keyword if platform is qemu.
refer - http://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Return%20From%20Keyword%20If